pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -9,16 +9,17 @@
   perl,
   texinfo,
   texinfo6,
-  yasm,
   nasm,
 
   # You can fetch any upstream version using this derivation by specifying version and hash
   # NOTICE: Always use this argument to override the version. Do not use overrideAttrs.
   version, # ffmpeg ABI version. Also declare this if you're overriding the source.
   hash ? "", # hash of the upstream source for the given ABI version
-  source ? fetchgit {
-    url = "https://git.ffmpeg.org/ffmpeg.git";
-    rev = "n${version}";
+  source ? fetchFromGitea {
+    domain = "code.ffmpeg.org";
+    owner = "FFmpeg";
+    repo = "FFmpeg";
+    tag = "n${version}";
     inherit hash;
   },
```

Comment:
Is this actually the new upstream canonical source? My understanding was that that are accepting contributions via Forgejo now, but that it wasn’t yet the sole repository.

It probably doesn’t matter either way, anyway; my only worry is whether they’re expecting the Forgejo tarball fetch path here to be load‐bearing infrastructure.

Reply:
I thought I read that they are switching over to it, but reading the release notes again, yeah, it just mentions that they are also accepting contributions over it, nothing about changing over to it.